EngineLayer/GlycoSearch/ModBox.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace EngineLayer
+{
+    public class ModBox
```

Comment:
i'm not sure but based on your above comments it seems like `ModBox` is referring to a list of possible modifications on the peptide or at a particular residue. can you name this class something like `ModificationPossibilities` or something more obviously understandable?

Reply:
I prefer to keep it like this. I will comment it well.